addons/pos_restaurant/static/src/app/split_bill_screen/split_bill_screen.xml

```diff
@@ -27,16 +27,44 @@
                             <span class="subtotal text-success">
                                 <t t-esc="env.utils.formatCurrency(newOrderPrice)" />
                             </span>
+                             <span class="total">
+                                / <t t-esc="env.utils.formatCurrency(getTotalAmount())" />
+                            </span>
                         </div>
-                        <div class="pay-button">
-                            <div class="button btn btn-lg btn-primary py-3 py-lg-5 w-100" t-on-click="createSplittedOrder">
-                                <span>Split Order</span>
+                        <t t-if="!ui.isSmall">
+                            <div class="pay-button m-3">
+                                <t t-set="nbrProduct" t-value="getProductSelectedQuantity() " />
+                                <div class="button btn btn-lg py-3 w-100"
+                                     t-on-click="createSplittedOrder"
+                                     t-att-class="nbrProduct &gt; 0 ? 'btn-primary' : 'btn-disabled'">
+                                    <i class="oi oi-chevron-right me-2" />
+                                    <span>Split Order</span>
+                                    <br/>
+                                    <span class="fs-6">
+                                        <t t-esc="nbrProduct" /> product<span t-if="nbrProduct &gt; 1">s</span> selected
+                                    </span>
+                                </div>
                             </div>
-                        </div>
+                        </t>
+                        <t t-if="ui.isSmall">
+                            <div class="pay-button d-flex">
+                                <t t-set="nbrProduct" t-value="getProductSelectedQuantity() " />
+                                <div class="btn-switchpane validation-button btn btn-secondary flex-fill py-3 gap-0 rounded-0 fw-bolder fs-1 w-50"
+                                     t-on-click="createSplittedOrder"
+                                     t-att-class="nbrProduct &gt; 0 ? 'btn-primary' : 'btn-secondary'">
+                                    <span>Split</span>
+                                    <div class="fs-6 m-0">
+                                        <t t-esc="nbrProduct" /> product<span t-if="nbrProduct &gt; 1">s</span> selected
+                                    </div>
+                                </div>
+                                <div class="btn-switchpane validation-button btn btn-secondary flex-fill py-3 rounded-0 fw-bolder fs-1 w-50 flex-align-center" t-on-click="back">
+                                    Back
+                                </div>
+                            </div>
```

Comment:
Make these mobile view buttons to the buttons of payment screen and product screen.